default/default.ctml

```diff
@@ -51,12 +51,25 @@
                 <ul id="dependencies" iterate="dependencies"><li><a class="external" lquery="(text (asdf:component-name *)) (attr :href (system-field 'homepage *))"></a></li></ul>
               </div>
             </c:when>
-            <c:when test="author">
-              <div class="row">
-                <label for="author">Author:</label>
-                <a id="author" href="#" lquery='(attr :href (when (extract-author-email author) (format NIL "mailto:~a" (extract-author-email author)))) (text (extract-author-name author))'>author</a>
-              </div>
-            </c:when>
+            <c:if test="(stringp author)">
+              <c:then>
+                <div class="row">
+                  <label for="author">Author:</label>
+                  <a id="author" href="#" lquery='(attr :href (when (extract-author-email author) (format NIL "mailto:~a" (extract-author-email author)))) (text (extract-author-name author))'>author</a>
+                </div>
+              </c:then>
+              <c:else>
+                <c:when test="author">
+                  <div class="row">
+                    <label id="author-label" for="authors">Authors:</label>
```

Comment:
It would be better not to have this ID, and instead make the entire `div class="row" id="author"` in both branches.